docs/src/manuals/meta-specification.md

```diff
@@ -57,35 +63,185 @@ List of available options:
 First, let's start with an example:
 
 ```julia
-meta = @meta begin 
+my_meta = @meta begin 
     GCV(x, k, w) -> GCVMetadata(GaussHermiteCubature(20))
 end
 ```
 
-This meta specification indicates, that for every `GCV` node in the model with `x`, `k` and `w` as connected variables should use the `GCVMetadata(GaussHermiteCubature(20))` meta object.
+This meta specification indicates that for every `GCV` node in the model with `x`, `k` and `w` as connected variables should use the `GCVMetadata(GaussHermiteCubature(20))` meta object.
 
 You can have a list of as many meta specification entries as possible for different nodes:
 
 ```julia
-meta = @meta begin 
+my_meta = @meta begin 
     GCV(x1, k1, w1) -> GCVMetadata(GaussHermiteCubature(20))
-    GCV(x2, k2, w3) -> GCVMetadata(GaussHermiteCubature(30))
-    NormalMeanVariance(y, x) -> MyCustomMetaObject(arg1, arg2)
+    AR() -> ARMeta(Univariate, 5, ARsafe())
+    MyCustomNode() -> MyCustomMetaObject(arg1, arg2)
 end
```

Comment:
I have personal preferences, but I would like to see a model to map to this meta.